core/domain/blog_services_test.py

```
@@ -20,26 +20,23 @@
 from core import feconf, utils
 from core.domain import (
     blog_domain,
     blog_services,
-    search_services,
     user_services,
 )
 from core.platform import models
 from core.tests import test_utils
 
-from typing import Any, Dict, List, Tuple
+from typing import Any, Dict, List
```

Ruff (0.16.1)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)